livekit-plugins/livekit-plugins-playai/livekit/plugins/playai/tts.py

```diff
@@ -0,0 +1,313 @@
+from __future__ import annotations
+
+import asyncio
+import logging
+import os
+from dataclasses import dataclass
+
+from livekit import rtc
+from livekit.agents import (
+    DEFAULT_API_CONNECT_OPTIONS,
+    APIConnectionError,
+    APIConnectOptions,
+    tokenize,
+    tts,
+    utils,
+)
+from pyht import AsyncClient as PlayHTAsyncClient  # type: ignore
+from pyht.client import Format, Language, TTSOptions  # type: ignore
```

Comment:
I don't think we want to ignore type checks for the whole pyht package?

Reply:
their package is not typed